frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -174,6 +188,12 @@
 <Page wide>
   <div class="space-y-4">
     {#if project}
+      {#if migrationStatus === ProjectMigrationStatus.Migrating}
+        <div class="alert alert-warning">
+          <span class="i-mdi-alert text-2xl" />
+          <span>This project is currently being migrated. Some features may not work as expected.</span>
```

Comment:
This should be translated.
Also, I think we can safely make it more specific and concrete. E.g. "Send/Receive will not work until the migration is complete."

I see now that you just copied the text from the notification, but I'd still appreciate a minor rewrite.

Reply:
I think we can skip translation. This code will be removed once all projects have been migrated and I don't think we'll have any other languages by then.

Reply:
Will skip translation here too.